ios/MarigoldSDKReactNativeTests/RNEngageBySailthruTests.m

```diff
@@ -393,6 +395,22 @@ -(void)logAbandonedCart:(NSDictionary *)purchaseDict resolver:(RCTPromiseResolve
             [[check should] equal:error];
         });
     });
+    
+    context(@"the clearAttributes:rejecter: method", ^{
```

Comment:
Could we expand on these tests to make sure the error handling/callbacks are being used correctly? The `profileVars` tests are a good example